modules/service/src/test/scala/lucuma/odb/graphql/DatabaseOperations.scala

```diff
@@ -1264,6 +1266,24 @@ trait DatabaseOperations { this: OdbSuite =>
       .use(_.prepareR(af.fragment.query(target_id)).use(_.unique(af.argument)))
   }
 
+  def createCalibrationProgram(
+    calibrationRole: CalibrationRole,
+    description: Description
+  ): IO[Program.Id] = {
+    val af = sql"""
+      insert into t_program (
+        c_calibration_role,
+        c_library_desc
+      )
+      select
+        $calibration_role,
+        $text
+      returning c_program_id
+    """.apply(calibrationRole, description.value)
+    FMain.databasePoolResource[IO](databaseConfig).flatten
+      .use(_.prepareR(af.fragment.query(program_id)).use(_.unique(af.argument)))
+  }
```

Comment:
There isn't a service method that would do this?

Reply:
Not really as it is not expected to create system programs programmatically. For example, it isn't allow in the API.

Reply:
Wouldn't the programmatic update use the service?  If we can, I think we want to funnel all these database manipulations into service code.